_layouts/note.html

```diff
@@ -22,7 +22,7 @@ <h3 style="margin-bottom: 1em">Notes mentioning this note</h3>
       <div style="display: grid; grid-gap: 1em; grid-template-columns: repeat(1fr);">
       {% for backlink in page.backlinks %}
         <div class="backlink-box">
-        <a class="internal-link" href="{{ backlink.url }}{%- if site.use_html_extension -%}.html{%- endif -%}">{{ backlink.title }}</a><br>
+        <a class="internal-link" href="{{site.baseurl}}{{ backlink.url }}{%- if site.use_html_extension -%}.html{%- endif -%}">{{ backlink.title }}</a><br>
```

Comment:
Same here - for formatting consistency, we could insert spaces around `site.baseurl`:

```suggestion
        <a class="internal-link" href="{{ site.baseurl }}{{ backlink.url }}{%- if site.use_html_extension -%}.html{%- endif -%}">{{ backlink.title }}</a><br>
```